CHANGES.rst

```diff
@@ -49,6 +49,9 @@ flatfield
 - Propagate uncertainty from FFLAT, SFLAT and DFLAT flat fields into science
   ERR array and VAR_FLAT array for NIRSpec spectroscopic modes.  [#3538]
 
+- Fix bug in NIRSpec fore-optics flat err array interpolation.  Make default
+  value of flat error array zero for wavelength ranges not covered. [#3564]
+
```

Comment:
I don't think a change log is needed - this hasn't been released yet.

Reply:
I agree. I would consider it to just be part of the overall process of implementing flat-field variance for NIRSpec.